great_expectations/experimental/datasources/pandas_datasource.py

```diff
@@ -72,18 +74,18 @@ class PandasDatasourceError(Exception):
     ParquetAsset = _FilesystemDataAsset
 
 
-class PandasDatasource(Datasource):
+class _PandasDatasource(Datasource):
     # class attributes
     asset_types: ClassVar[List[Type[DataAsset]]] = list(_ASSET_MODELS.values())
 
     # instance attributes
-    type: Literal["pandas"] = "pandas"
-    name: str
+    type: str = pydantic.Field("_pandas")
```

Comment:
If this is a `_private` datasource it shouldn't need a `type` field as it won't be "registered" by the `MetaDatasource` `__new__` method.

Reply:
If I change this to just `type: str` I get: `great_expectations.experimental.datasources.sources.TypeRegistrationError: '_PandasDatasource' is missing a 'type' attribute with an assigned string value`

Also, I had to change the backticks in that error message to single quotes to show it above. Do we want backticks in error messages?

Reply:
Also note I had to ignore `_PandasDatasource` in `tests/experimental/datasources/test_schemas.py`.

Reply:
Per our offline conversation @Kilo59 will handle this issue in the registry.